taskqueues.lua

```diff
@@ -674,6 +674,7 @@ local anyLvl2ShipYard = {
 	Lvl2ShipBreakthrough,
 	Lvl2ShipMerl,
 	Lvl2ShipAssist,
+	Lv2SubWar,
```

Comment:
it's easier to find things in the future if the naming conventions are consistent. function should be named `Lvl2SubWar`